doc/user/less-simple.xml

```diff
@@ -463,30 +463,30 @@ Program('program', src_files)
     &SCons; also allows you to identify
     the output file and input source files
     using Python keyword arguments.
-    The output file is known as the
-    <emphasis>target</emphasis>,
-    and the source file(s) are known (logically enough) as the
-    <emphasis>source</emphasis>.
+    The output file keyword is
```

Comment:
I don't think keyword is additive.

Reply:
didn't quite understand this comment.

Reply:
Okay, I think I see.  Simplified, it's now "let the example explain".

Reply:
done.